### DISCUSSION TITLE
RQST icon color on >0 state

### DISCUSSION THREAD

**Discussion by GH2user** [marcokreeft87/room-card#52] [Ideas]
Hi,

I have added a lot of entities which have either a state of 0 or bigger when running.
To emphasise the unit is running I have the icon color set to the active state using icon conditions.

 - entity: sensor.plug_dryer_power
    name: dryer
    icon:
      conditions:
        - icon: mdi:tumble-dryer
          condition: equals
          value: 0
        - icon: mdi:tumble-dryer
          condition: above
          value: 0
          styles:
            color: var(--paper-item-icon-active-color)
    show_state: true
    format: precision1

It would be great if we had a boolean to do this automatically;        **color_nonzero: true**
Just to remove lots of repetitive code in the card.

 - entity: sensor.plug_dryer_power
    name: dryer
    color_nonzero: true
    show_state: true
    format: precision1

Thanks.
PS code-tags put it all on one line unfortunately

**Comment by marcokreeft87** [comment]
Sorry this is too much custom work. Maybe in the future, but this won't be in in the near future.

**Comment by GH2user** [comment]
OK understand. As I have no clue how much work or if even possible just thought I raise the question.

**Reply by marcokreeft87** [reply]
No problem,  you can always ask right? :)

**Comment by marcokreeft87** [comment]
Hmm maybe I can make something you would like. Like templating. So you can put the repetitive yaml in a template. But first I need to cover the card with unit tests.

**Comment by GH2user** [comment]
I think that would be great and many more people will think so I suspect.
The custom:button-card has templating and it really can be useful.
No hurry and if I need to do some testing just let me know,

**Comment by marcokreeft87** [comment]
@GH2user check out version 1.5.0 and see if this works for you

https://github.com/marcokreeft87/room-card/wiki/Configuration#icon-templating

**Comment by marcokreeft87** [comment]
@GH2user check v1.06.00 I think you will like this :) 

https://github.com/marcokreeft87/room-card/wiki/Configuration#templates
